Integrations/integration-Moloch.yml

```diff
@@ -48,6 +48,12 @@ script:
 
     var pcap_command = 'moloch_sessions_pcap';
 
+    var other_fields = "ipProtocol";
```

Comment:
Please add a comment about why we add those fields

Reply:
Done, will push the changes when the build will be green